Main-bigint/Main-bigint/bigint.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+class BigInteger
+{
+private:
+    unsigned char bigint[16];
+    void clear();
+    void overflowWarning(); 
+
+public:
+    BigInteger();
+    BigInteger(std::string str);
+    BigInteger(const BigInteger &another);
+
+    unsigned char* getValue();
+    unsigned char getChar(const int& indexOfArray);
+    void setBit(const int& indexOfArray, const int& indexInArray); // maybe not used
+    void setChar(const int& indexOfArray, char value);
+
+    void strToBigInt(std::string str);
+    std::string bigIntToStr();
+    bool isNegative();
+    bool isNegative(BigInteger number);
```

Comment:
Why do we have this method? If I want to check a BigInt is negative with this method, the code is
```
BigInt number("432123");
bool isNegative = number.isNegative(number);
```
I think this use is weird. This method should be static. Or we only need `bool isNegative();`